.github/workflows/nightly-chart-and-image-publish.yaml

```diff
@@ -111,10 +115,10 @@ jobs:
         with:
           version: 3.8.0
       - name: Build Helm chart
-        run: make release-chart RELEASE_TAG=${{ env.TAG }}
+        run: make release-chart RELEASE_TAG=${{ env.TAG }} CONTROLLER_IMAGE_VERSION=${{ env.TAG }} 
```

Comment:
Does this controller image version do anything here?

Reply:
Yes, we want to set imageVersion in values.yaml

Reply:
The tag is used for that, this variable is not present in the code

Reply:
https://github.com/rancher-sandbox/rancher-turtles/blob/main/Makefile#L480, currently this value is empty and chart doesn't install